### PR TITLE
Verify that the restore of Kubevirt VM is successful and the VM pods are running on the designated Kubevirt nodes

### PR DESCRIPTION
**Pull request by atrivedi-px**
**What this PR does / why we need it**:
Verify that the restore of Kubevirt VM is successful and the VM pods are running on the designated Kubevirt nodes

**Which issue(s) this PR fixes** (optional)
Closes #PB-5794

**Special notes for your reviewer**:
Aetos Run: https://aetos.pwx.purestorage.com/resultSet/testSetID/556395

### CODE REVIEW

File: tests/backup/backup_kubevirt_test.go [hunk 1]
```diff
@@ -719,3 +721,290 @@ var _ = Describe("{KubevirtVMSshTest}", func() {
...
+			}
+		}
+
+		// Generate random labels to be added to node and VM as node selector
+		nodeSelectorPresent[fmt.Sprintf("node_selector_%s", RandomString(4))] = fmt.Sprintf("value_%s", RandomString(6))
+		// Generate random labels to be added to node and VM as node selector
+		nodeSelectorNotPresent[fmt.Sprintf("node_selector_%s", RandomString(4))] = fmt.Sprintf("value_%s", RandomString(6))
+	})
+
+	It("Verify backup and restore of Kubevirt VMs in different states", func() {
```
Comment (vpinisetti-px) [inline tests/backup/backup_kubevirt_test.go:781]: Can you correct this string as per the testcase

Comment (atrivedi-px) [reply tests/backup/backup_kubevirt_test.go]: Done

File: tests/backup_helper.go [hunk 2]
```diff
@@ -7271,3 +7277,50 @@ func ScaleApplicationToDesiredReplicas(namespace string) error {
 
 	return nil
 }
+
+// AddNodeSelectorToApps applies node selector to the virtual machine
```
Comment (mkoppal-px) [inline tests/backup_helper.go:7281]: Please fix the doc string

Comment (atrivedi-px) [reply tests/backup_helper.go]: Done

**Review by vpinisetti-px — APPROVED** [review]
LGTM

**Pull request merged**